bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiControllerThingHandler.java

```diff
@@ -138,14 +138,14 @@ private void start(final UniFiController uc) {
             uc.start();
             startRefresh = true;
         } catch (final UniFiCommunicationException e) {
-            updateStatus(OFFLINE, COMMUNICATION_ERROR, STATUS_DESCRIPTION_COMMUNICATION_ERROR);
+            updateStatus(OFFLINE, COMMUNICATION_ERROR, STATUS_DESCRIPTION_COMMUNICATION_ERROR + e.getMessage());
```

Comment:
Will this work without surrounding `[]`? Like here:

https://github.com/openhab/openhab-addons/blob/80094b5e9d293dfb73c81377325da13172d6baaf/bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java#L107-L108

https://github.com/openhab/openhab-addons/blob/80094b5e9d293dfb73c81377325da13172d6baaf/bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/i18n/danfossairunit.properties#L84

Just asking, since I'm not aware of this syntax separating I18N text reference and parameter value with just a space, and don't have time to test it right now.

Reply:
Yes it works because core uses handles this in a lenient way. See https://github.com/openhab/openhab-core/blob/dd756b8e2505c6cf9c7555ec7cb73965fd3a5328/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/i18n/ThingStatusInfoI18nLocalizationService.java#L174

I had looked in the code, but could not directly see what the intended usage was. However I've now changed it to use the string array notation as that seems to be what is the official syntax it should be.

Reply:
Thanks for the reference and explanation.